builders/get-started/eth-compare/transfers-api.md

```diff
@@ -25,28 +25,42 @@ Token持有者有两种方式来启动Moonbeam上的余额转账功能。一方
- - **Substrate功能** —— 一些原生Substrate功能可以创建Extrinsic，将Token发送至一个地址。例如，[Treasury](/learn/features/treasury/){target=_blank}可以创建一个Extrinsic，如`treasury.proposeSend`，这将触发**一个或多个**`balances.Transfer`事件
- - **Ethereum转账** —— 这将创建一个`ethereum.transact`Extrinsic，为一个空白输入值。这将触发**一个**`balances.Transfer`事件
- -  **通过智能合约进行以太坊转账** —— 这将创建一个`ethereum.transact`Extrinsic，多个数据成为输入值。这将触发**一个或多个**`balances.Transfer`事件
+- **Substrate转账** — 这将创建一个Extrinsic，`balances.transfer`或`balances.transferKeepAlive`。这将触发**一个**`balances.Transfer`事件
+- **Substrate功能** — 一些原生Substrate功能可以创建Extrinsic，将Token发送至一个地址。例如，[Treasury](/learn/features/treasury/){target=_blank}可以创建一个Extrinsic，如`treasury.proposeSend`，这将触发**一个或多个**`balances.Transfer`事件
+- **Ethereum转账** — 这将创建一个`ethereum.transact`Extrinsic，为一个空白输入值。这将触发**一个**`balances.Transfer`事件
+- **通过智能合约进行以太坊转账** — 这将创建一个`ethereum.transact`Extrinsic，多个数据成为输入值。这将触发**一个或多个**`balances.Transfer`事件
 
 上述所有场景都将能有效地进行原生代币转账。监控它们最简单的方法就是通过`balances.Transfer`事件。
 
-## 使用Substrate API来监控所有余额转账 {: #monitor-transfers }
+## 监控原生Token余额转账 {: #monitor-transfers }
+
+以下代码示例将演示如何使用[Polkadot.js API库](https://polkadot.js.org/docs/api/start){target=_blank}或[Substrate API Sidecar](https://github.com/paritytech/substrate-api-sidecar){target=_blank}监听通过Substrate或Ethereum API发送的两种类型的原生Token转账。下方代码片段仅用于演示目的，请将其进行调整并进一步测试后再用于生产环境。
+
+### 使用Polkadot.js API {: #using-polkadotjs-api }
 
 [Polkadot.js API程序包](https://polkadot.js.org/docs/api/start){target=_blank}为开发人员提供了一种使用Javascript与Substrate链相交互的方式。
 
-以下代码片段使用[`subscribeFinalizedHeads`](https://polkadot.js.org/docs/substrate/rpc/#subscribefinalizedheads-header){target=_blank}订阅新的已确认区块头，并循环访问之中的每个事件。然后，检查是否与一个`balances.Transfer`事件对应。如果是，这将提取一个转账的`from`、`to`以及`amount`并显示在控制台上。请注意，`amount`是以最小的单位（Wei）来显示的。您可以在他们的[官方文档网站](https://polkadot.js.org/docs/substrate/rpc){target=_blank}找到关于Polkadot.js和Substrate JSON RPC的所有可用信息。
+以下代码片段使用[`subscribeFinalizedHeads`](https://polkadot.js.org/docs/substrate/rpc/#subscribefinalizedheads-header){target=_blank}订阅最新确认的区块头，并循环访问之中的每个事件。然后，检查是否与一个`balances.Transfer`事件对应。如果是，这将提取一个转账的`from`、`to`以及`amount`并显示在控制台上。请注意，`amount`是以最小的单位（Wei）来显示的。您可以在他们的[官方文档网站](https://polkadot.js.org/docs/substrate/rpc){target=_blank}找到关于Polkadot.js和Substrate JSON RPC的所有可用信息。
 
 --8<-- 'code/vs-ethereum/transfers-api/balance-event.md'
 
 此外，您可以在[此脚本](https://gist.github.com/crystalin/b2ce44a208af60d62b5ecd1bad513bce){target=_blank}中找到更多余额转账相关具体案例的代码片段。
+
+### 使用Substrate API Sidecar {: #using-substrate-api-sidecar }
+
+开发者也可以使用[Substrate API Sidecar](https://github.com/paritytech/substrate-api-sidecar){target=_blank}检索Moonbeam区块并监控通过Substrate和Ethereum API发送的交易。Substrate API Sidecar是一种REST API服务，用于与使用Substrate框架构建的区块链交互。
+
+以下代码片段使用Axios HTTP客户端查询Sidecar端点`/blocks/head`(https://paritytech.github.io/substrate-api-sidecar/dist/){target=_blank}以获取最新确认的区块头。然后，在EVM和Substrate API级别解码原生Token转账的`from`、`to`、`value`、`tx hash`和`transaction status` 区块。
+
+--8<-- 'code/vs-ethereum/transfers-api/sidecar-transfer.md'
```

Comment:
There's some unnecessary white space here I believe:
`|| extrinsic.   args.transaction.eip1559 &&`

Reply:
opened PR in EN repo for this: https://github.com/moonbeam-foundation/moonbeam-docs/pull/709